infra/src/main/kotlin/fi/fta/geoviite/infra/geocoding/Geocoding.kt

```diff
@@ -200,6 +222,10 @@ data class GeocodingContext(
         val endAddress = alignment.end?.let(::getAddress)?.first
         return if (startAddress == null || endAddress == null || address !in startAddress..endAddress) {
             null
+        } else if (startAddress.isSame(address)) {
```

Comment:
Testeissä huomasin että pyöristysvirheiden takia voi joskus käydä niin että aivan raiteen päässä olevalla osoitteella ei voikaan geokoodata, sillä projektioviivan seuranta takaisinpäin ei välttämättä ihan osukaan raiteelle. Palautetaan tässä siis vain alku-/loppupiste jos niitä pyydettiin.

Reply:
Onko näissä tarkoituksella `!!`, vai onko unohdusta? Mehän ollaan koitettu päästä tuon käytöstä eroon, niin jos tämän voisi tehdä ei-nurinkurisesti jotenkin muuten, niin voisi olla kiva

Reply:
Ihan totta kyllä. Kyllä tuon saa pyöräytettyä jotenkin pois, korjaan..